README.md

```diff
@@ -8,35 +6,56 @@ Terraform module to create a Function App which automates certificate management
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.14.6 |
-| azurerm | =2.58.0 |
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.15.1 |
+| <a name="requirement_azurerm"></a> [azurerm](#requirement\_azurerm) | =2.58.0 |
 
 ## Providers
 
 | Name | Version |
 |------|---------|
-| azuread | n/a |
-| azurerm | =2.58.0 |
-| azurerm.dnszone | =2.58.0 |
-| azurerm.privatezone | =2.58.0 |
+| <a name="provider_azuread"></a> [azuread](#provider\_azuread) | n/a |
```

Comment:
same here?

Reply:
There is bug in the code. Links aren't valid ones. I am using the latest version of terraform-docs.